content/en/logs/log_collection/nodejs.md

```diff
@@ -99,23 +99,19 @@ logger.info('Hello log with metas',{color: 'blue' });
 {{% /tab %}}
 {{< /tabs >}}
 
-Check the content of the `<FILE_NAME>.log` file to see that Winston already took care of logging everything in JSON:
+Check the content of the `<FILE_NAME>.log` file to confirm that Winston is logging in the JSON format:
 
 ```json
 {"level":"info","message":"Hello simple log!","timestamp":"2015-04-23T16:52:05.337Z"}
 {"color":"blue","level":"info","message":"Hello log with metas","timestamp":"2015-04-23T16:52:05.339Z"}
 ```
 
-## Connect your service across logs and traces
-
-If APM is enabled for this application, connect your logs and traces by automatically adding trace IDs, span IDs,
-`env`, `service`, and `version` to your logs by [following the APM NodeJS instructions][3].
-
-**Note**: If the APM tracer injects `service` into your logs, it overrides the value set in the agent configuration.
-
 ## Configure your Datadog Agent
 
-Create a `nodejs.d/conf.yaml` file in your `conf.d/` folder with the following content:
+Once [log collection is enabled][6], set up [custom log collection][7] to tail your log files and send them to Datadog.
```

Comment:
```suggestion
Once [log collection is enabled][6], set up [custom log collection][7] to tail your log files and send new logs to Datadog.
```

Reply:
This one is up to you, I just remember having customers that were confused about why old logs were not being sent, so thought I'd add this in as a clarification